docs/en/userguide/form-submissions.md

```diff
@@ -85,3 +85,114 @@ This decides whether to send the email based on two options
 * Select the field which you want the custom rule to apply to
 * Select the condition the field must follow
 * enter for the condition (the 'is blank' and 'is not blank' conditions do not require any text)
+
+## File Upload Field
+
+[note]
+This functionality is specifically included in Silverstripe core functionality 4.6 and above, 
+[silverstripe-userforms](https://github.com/silverstripe/silverstripe-userforms/) 5.3 and above and is also included in 
+the module [silverstripe-elemental-userforms](https://github.com/dnadesign/silverstripe-elemental-userforms).
+[/note]
+
+Files and other data that is uploaded by your website visitors should be handled with care to reduce the risk of exposing sensitive data. 
+Files which are uploaded through a form inherit the permissions of the folder they are being placed into, so it's important that you 
+ensure the folder you select can only be viewed by appropriate parties. Folder permissions can be managed within the Files area.
+
+File permissions do not affect the Draft/Modified/Published state of files. Use both file permissions and Draft states 
+together to ensure your files remain safe. For example, in case they are accidentally published. 
+
+## File indicators
+
+Files have a number of icons which can help to identify the original source of a file and whether caution should be taken when using it. 
+
+**Form submission:** ![Address card icon](_images/address-card.png)
+
+This indicates a file is associated with a form submission, this file could contain personal identifiable information and care should be taken so that it is not leaked.
```

Comment:
```suggestion
This indicates a file is associated with a form submission. This file has not been uploaded to the CMS by a Content Manager and may contain sensitive information. Care should be taken so that it is not published and leaked.
```

I think PII is only one example of sensitive information.